modin/core/storage_formats/pandas/parsers.py

```diff
@@ -671,11 +699,9 @@ class PandasParquetParser(PandasParser):
 """,
     )
     def parse(files_for_parser, **kwargs):
-        from pyarrow.parquet import ParquetFile
-
         columns = kwargs.get("columns", None)
         storage_options = kwargs.pop("storage_options", {}) or {}
-
+        engine = kwargs.pop("engine")
```

Comment:
if "engine" is popped here, it won't be passed into [this following line](https://github.com/modin-project/modin/pull/4807/files#diff-2ad1882ffe66014d5097154a11168dde541dbf7d8fed468663b8098591912bf6L682)  

`pandas.read_parquet(files_for_parser, **kwargs)` for a "single_worker_read".

Why do we not pass `engine` here, if `pandas.read_parquet` supports that kwarg?

Reply:
You are right. That's a good catch, I should just pass it along.